opnode/rollup/driver/state.go

```diff
@@ -130,6 +141,38 @@ func (s *state) loop() {
 		// case <-l2Poll.C:
 		case <-s.done:
 			return
+		case <-l2BlockCreation.C:
+			// 0. Make sure to only do this as a sequencer
+			if !s.sequencer {
+				continue
+			}
```

Comment:
To optionally ignore channel events there's a better trick: set the channel to `nil` if it should not be listened from. The select statement will not read from the `nil` channel.